sycl/doc/GetStartedWithSYCLCompiler.md

```diff
@@ -16,39 +19,129 @@ to install.
 SYCL support: follow the instructions under
 [SYCL* Compiler and Runtimes](https://github.com/intel/llvm/releases/tag/2019-07)
 
+### Get the required tools:
```

Comment:
the build description refers to ninja so please include that as required tool

Reply:
'ninja' is part of MSVC-2017 and MSVC-2019. MSVC is already mentioned in the required tools.
That was one of the reasons why I used 'ninja'.  Using 'make' would require additional downloads/installations.
The 2nd reasons was - ninja is much faster for incremental builds.

Reply:
I didn't know that, nice.  So it looks like cmake is also part of the VS2017 installation?

Reply:
You would need: 'python.exe' for msvc-2019
'python.exe' + 'cmake.exe' for msvc-2017

Reply:
@mibintc @v-klochkov we require CMake 3.2+. I'd rather go with something like `cmake --build . --target sycl-toolchain` and never care about build tool again.

Reply:
Alex, from your comment I don't understand what you suggest.
Do you suggest only saying explicitly in the doc that cmake must be 'cmake 3.2 or later'?
Or also change the 'cmake' command use below at the section explaining how to build compiler?

Reply:
@v-klochkov the latter. I think it would be better to make our instructions generator-agnostic as much as possible. Using `cmake` for build is considered best practice and shows people how to use whatever generator they have/like/etc.

Reply:
I still do not understand what you suggest. May need some time for questions and experiments with the new approach.
For now I suggest merging the current version and update the document in a new patch.